crates/matrix-sdk-appservice/src/lib.rs

```diff
@@ -29,6 +29,16 @@
 //! The crate is in an experimental state. Follow
 //! [matrix-org/matrix-rust-sdk#228] for progress.
 //!
+//! The crate relies on the registration being always in sync with the actual
+//! registration used by synapse. That's because it's required for the access
+//! tokens and because membership states for virtual users are determined based
+//! on the registered namespace.
+//!
+//! ## Warning
```

Comment:
I think we can remove the "Warning" heading here, as it's just an additional remark.

Reply:
Done